CHANGELOG.md

```diff
@@ -23,6 +24,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Do not display all the items that we restored at the end if there are more than 15. You can override this with `--verbose`.
 
+### Known Issues
+- Changes to attachments in instances of recurring events compared to the series master aren't restored
```

Comment:
separate PR: we should put this on the website docs known issues, too.

Reply:
Should be available already [here](https://github.com/alcionai/corso/blob/b560a89b5e6d22737f7e70796c5c083b51bdb224/website/docs/support/known-issues.md#L30).